NEWS.md

```diff
@@ -2,15 +2,20 @@ Hit tab to unselect buttons and scroll through the text using UP/DOWN or
 PGUP/PGDN. All announcements are stored in `/opt/openhabian/docs/CHANGELOG.md`
 for you to lookup.
 
+## Electric Vehicle Charge Controller ## April 6, 2022
+EVCC controls charging of your EV. It supports many wallboxes and vehicles.
+While there's some overlap with OH, EVCC has a two-way API and users can 
+selectively combine the best of both worlds.
+
 ## Raspberry Pi OS 64bit support ## April 1, 2022
```

Comment:
Remember to remove any announcements older than the last 3, and add any new ones to `docs/CHANGELOG.md`

Reply:
done